goalgorilla/open_social
# Issue #3151112 by rolki: I wanna be able to use the latest version of Infinite Scroll

## Problem
There are two issues:
- Not the latest version of Infinite Scroll
- Patch that we have currently creates another issue, see [https://www.drupal.org/project/drupal/issues/2821962#comment-13704038](https://www.drupal.org/project/drupal/issues/2821962#comment-13704038)

## Solution
- Update `infinite_scroll` to the latest version (1.9.0)
- Update existing patch with new one

## Issue tracker
* https://www.drupal.org/project/social/issues/3151112
* https://getopensocial.atlassian.net/browse/FOR-293

## How to test
- Install and enable the [social_scroll](https://bitbucket.org/goalgorilla/social_scroll/src/main/) extension
- Go to `admin/config/opensocial/infinite-scroll`
- Ensure that selected some views
- Go to this views page and try to use Infinite scroll instead of default pager
- Also go to some view that has exposed filters and check if everything  works there

## Screenshots
![image](https://user-images.githubusercontent.com/50984627/139425466-1511eff3-b717-4201-884d-f7ec6d9dc856.png)
![image](https://user-images.githubusercontent.com/50984627/139425519-c5dbe5b5-5661-4136-9b66-e556cf811c17.png)
